### PR TITLE
Finesse sign up.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review.

### CODE REVIEW

File: shared/settings/account/add-modals.tsx [hunk 1]
```diff
@@ -88,8 +89,12 @@ export const Email = () => {
           searchable={searchable}
           onChangeSearchable={onChangeSearchable}
           onContinue={onContinue}
-          icon={
-            <Kb.Icon type={Styles.isMobile ? 'icon-email-add-96' : 'icon-email-add-64'} style={styles.icon} />
+          iconType={
+            Styles.isMobile
```
Comment (buoyad) [inline shared/settings/account/add-modals.tsx]: `Styles.isMobile && Platform.isLargeScreen ? "big" : "small"`

Comment (adamjspooner) [reply shared/settings/account/add-modals.tsx]: Is that why the dimension styles were on them?

Comment (buoyad) [reply shared/settings/account/add-modals.tsx]: Yes

Comment (buoyad) [reply shared/settings/account/add-modals.tsx]: (I'm fine with removing them, rather do it in `Icon` than add more plumbing to make it work with your changes here)

Comment (adamjspooner) [reply shared/settings/account/add-modals.tsx]: Works for me.

**Pull request merged**